lisp/cl-ppcre/Portfile

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           common_lisp 1.0
+
+github.setup        edicl cl-ppcre 2.1.1 v
+revision            1
```

Comment:
I know this is nit-picky, but set your revision to zero.  ;-)

No worries though, I know it's tough when you're juggling this many ports in one PR!

Reply:
@mascguy it isn't. I've moved original port and removed variants, and heavy reworked it ;)

Reply:
Ah gotcha, no worries!